api/cuckoo_service.py

```diff
@@ -49,29 +53,27 @@ def upload_to_cuckoo(tracker_id, app):
     except Timeout:
         return False, "Connection timed out."
 
-def start_cuckoo_monitor(tracker_id, app):
-    log_manager = LogManager(tracker_id, app)
-    
-    try:
-        additional_data = {"error_message": "monitor start"}
-        log_manager.update_log_stage("monitor", additional_data)
-    except Exception as e:
-        print(f"Error during monitor log update: {e}")
+def start_cuckoo_monitor(tracker_id):
+    log_manager = LogManager(tracker_id)
+    print("hello")
+    additional_data = {"error_message": "monitor start"}
+    log_manager.update_log_stage("Cuckoo monitor started", additional_data)
 
-    # app_context = app.app_context()
-    # threading.Thread(target=check_cuckoo_status, args=(tracker_id, app, app_context), daemon=True).start()
+    threading.Thread(target=check_cuckoo_status, args=(tracker_id,), daemon=True).start()
 
-def check_cuckoo_status(tracker_id, app, app_context):
-    log_manager = LogManager(tracker_id, app)
-    with app_context:
-        LOG_FOLDER = app.config['LOG_FOLDER']
+def check_cuckoo_status(tracker_id):
+    print("mimo")
+    log_manager = LogManager(tracker_id)
+    with current_app.app_context():
+        LOG_FOLDER = current_app.config['LOG_FOLDER']
 
         try:
             log_path = os.path.join(LOG_FOLDER, f"{tracker_id}.json")
             with open(log_path, 'r') as log_file:
                 data = json.load(log_file)
                 task_id = data["task_id"]
         except FileNotFoundError:
+            print("filed")
             return False, "Doesn't find log file."
         
         while True:

```

Comment:
[Question]
Why `app_context`? You may just use `current_app` you import.

Reply:
Also, is it work? 
It it work, you may just put `current_app.app_context()` into your utility function. The route implemenration may not need too much `current_app.app_context()` declaration.